subroutines.sh

```diff
@@ -63,8 +138,6 @@ send_key() {
 }
 
 scan_bluetooth(){
```

Comment:
scan_bluetooth is not called anymore then? if yes, let's remove it
--> I also suggest to move all the presence subroutines in a dedicated .sh file?

Reply:
Might reuse it.... we need to split the big listen_to_ble function into smaller ones.